screen/send/SendDetails.tsx

```
@@ -129,186 +131,648 @@
+        }
+        setAmountUnit(BitcoinUnit.BTC);
+        setPayjoinUrl(pjUrl);
+      } catch (error) {
+        console.log(error);
+        presentAlert({ title: loc.errors.error, message: loc.send.details_error_decode });
+      }
+    } else if (routeParams.address) {
+      const { amount, amountSats, unit = BitcoinUnit.BTC } = routeParams;
+      // @ts-ignore: needs fix
+      setAddresses(value => {
+        if (currentAddress && currentAddress.address && routeParams.address) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)